test/utils/hasOnlyTheseExternalPayments.spec.js

```diff
@@ -0,0 +1,275 @@
+const { Testkit } = require('../../main')
+const { Network, Utils } = Testkit()
+
+describe('Response unit has only these payments', function () {
+	this.timeout(60000)
+
+	before(async () => {
+		this.network = await Network.create()
+		const genesis = await this.network.getGenesisNode().ready()
+		this.sender = await this.network.newHeadlessWallet().ready()
+
+		const { unit } = await genesis.sendBytes({ toAddress: await this.sender.getAddress(), amount: 1e9 })
+		await this.network.witnessUntilStable(unit)
+
+		this.asset_1 = (await this.sender.createAsset({
+			is_private: false,
+			is_transferrable: true,
+			auto_destroy: false,
+			issued_by_definer_only: true,
+			cosigned_by_definer: false,
+			spender_attested: false,
+			fixed_denominations: false,
+		})).unit
+		await this.network.witnessUntilStable(this.asset_1)
+	})
+
+	it('one base payment match', async () => {
+		const { unit } = await this.sender.sendBytes({ toAddress: 'WDZZ6AGCHI5HTS6LJD3LYLPNBWZ72DZI', amount: 185513 })
```

Comment:
Could you please provide strings '3W43U3SHKBVDUP7T7YOJOY5NM353HA5C' and 'WDZZ6AGCHI5HTS6LJD3LYLPNBWZ72DZI' with some kind of comment or something?
Or, I think, an even better solution for this would be to move them to const variables with some meaningful name
Kind of `const SMTH_ADDRESS = 'WDZZ6AGCHI5HTS6LJD3LYLPNBWZ72DZI'`

Reply:
constants are now used